rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -326,6 +326,16 @@ public Set<Class<?>> getClasses() {
         }
     }
```

Comment:
I would have preferred to add this comment to the beans.xml but I can't seem to do that (perhaps because it's empty).

Given the getClasses method is not in the `main` anymore I think, is it the addition of the empty `beans.xml` that cause something to trigger which scans the binary to find the `Coordinator` class and treat is as a resource instead?

Similarly, is that file the thing that somehow determines to treat `CoordinatorContainerFilter` as a singleton?

Reply:
not sure what you are referring to. Maybe misplaced comment.

Reply:
I will move it somewhere else, I wanted to add the comment to the beans.xml file but I couldn't spot a way to do that.

Reply:
I added an updated comment here https://github.com/jbosstm/narayana/pull/1966/files#r882565440